modules/modlauncher/src/main/java/org/gotti/wurmunlimited/modsupport/CreatureTemplateBuilder.java

```diff
@@ -371,8 +371,11 @@ public CreatureTemplate build() {
 				temp.setOnFire(onFire);
 				temp.setFireRadius(fireRadius);
 			}
+			
+			if (combatMoves != null) {
```

Comment:
You need to use unified code style as it helps with code readability. Please fix tabulation.